src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -283,12 +283,16 @@ void AbstractTableGenerator::generate_and_store() {
 
       const auto add_table = [&]() {
         Timer per_table_timer;
+        auto output = std::string{""};
         if (storage_manager.has_table(table_name)) {
```

Comment:
Good question. However, this code adaption within `tpch_table_generator.cpp` only resulted because i needed a (quick & dirty) way to somehow trigger the chunk::finalize method or at least gain knowledge about it. It is now obsolete and will be changed.